src/info/git/mod.rs

```diff
@@ -256,26 +257,21 @@ fn compute_diff_with_parent(
     );
 
     if let (tree_id, None) = parents {
-        tree_id
-            .object()?
-            .into_tree()
-            .changes()?
-            .track_path()
-            .track_rewrites(None)
-            .for_each_to_obtain_tree(&commit.tree()?, |change| {
-                let is_file_change = match change.event {
-                    Event::Addition { entry_mode, .. } | Event::Modification { entry_mode, .. } => {
-                        entry_mode.is_blob()
-                    }
-                    Event::Deletion { .. } | Event::Rewrite { .. } => false,
-                };
-                if is_file_change {
-                    let path = change.location;
-                    *change_map.entry(path.to_owned()).or_insert(0) += 1;
+        let old_tree = tree_id.object()?.into_tree();
+        let new_tree = commit.tree()?;
+        let changes = repo.diff_tree_to_tree(&old_tree, &new_tree, Options::default())?;
```

Comment:
Options needs to be tuned to do the equivalent of `track_rewrites(None)`. Otherwise performance is lower and `change.location()` won't actually count all paths, it would never see deletions if they were the source of a rename.

Reply:
You mean using `Options::default().with_rewrites(None)`?
It seems to be the default configuration https://github.com/GitoxideLabs/gitoxide/blob/ccd6525c1b39cea9c12a88d6ac4adf06d0df0b9c/gix/src/diff.rs#L27

Reply:
Ah, that's right! I'd suggest to be safe though and always explicitly state what's important in the options instead of leaving it to the implicit default.
Even I fell for it, and I am supposed to have written this `Default` implementation 😅.

Reply:
done